includes/common.php

```diff
@@ -994,11 +994,11 @@ function load_os(&$device)
 
     // Set type to a predefined type for the OS if it's not already set
     $loaded_os_type = Config::get("os.{$device['os']}.type");
-    if ((! isset($device['attribs']['override_device_type']) && $device['attribs']['override_device_type'] != 1) && array_key_exists('type', $device) && $loaded_os_type != $device['type']) {
```

Comment:
what about this `!= 1` check?

Reply:
`! $model->getAttrib('override_device_type')`  is the same.